scripts/generate.sh

```diff
@@ -243,7 +243,7 @@ dart () {
 
   openapi-generator-cli version-manager set 6.0.1
   openapi-generator-cli generate -i "${SPEC_FILE}" \
-    -g dart-dio \
+    -g dart \
```

Comment:
Shouldn't this be dart2?

Reply:
Great question!

Dart(1) was depreciated and now there's only Dart

Issue https://github.com/gibahjoe/openapi-generator-dart/issues/31

Change https://github.com/OpenAPITools/openapi-generator/pull/7568

Reply:
thank you for the explanation!